### ISSUE TITLE
[a11y]: Tooltip has 1 violation with an aria-hidden

### ISSUE THREAD

**Issue by hyungskim** [carbon-design-system/carbon#14921]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

v10.58.10 (carbon-component) 7.59.14 (carbon-component-react)

### React version

16.14

### Automated testing tool and ruleset

IBM Equal Accessibility Checker (latest one)

### Description

Tooltip has 1 violation with an aria-hidden

<img width="1865" alt="Screen Shot 2023-10-16 at 11 14 39 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/b7623cc0-fbb7-4d8f-b0e6-247797f11481">
<img width="1071" alt="Screen Shot 2023-10-16 at 11 14 58 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/ae544ea6-8dd8-4966-a7ce-84d3856e78a7">

### Reproduction/example

Reproducible using the Tooltip component

### Steps to reproduce

1. Use <Tooltip> in the UI page.
2. Run Accessibility checker against the page.

Hi, @guidari
I'm from Catalog UI IBM Knowledge Catalog. We've noticed 1 violation in the Tooltip component.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
